scripts/webpack/webpack.config.stats.ts

```diff
@@ -1,4 +1,4 @@
-import CleanWebpackPlugin from 'clean-webpack-plugin';
+import { CleanWebpackPlugin } from 'clean-webpack-plugin';
```

Comment:
TBH I don't think that this thing even works. @miroslavstastny is it still collecting stats?

Reply:
dunno about before, but now it should :D